server/blob/blob-objectstorage/src/test/java/org/apache/james/blob/objectstorage/swift/SwiftKeystone3ObjectStorageBlobsDAOBuilderTest.java

```diff
@@ -78,7 +77,7 @@
     @BeforeEach
     void setUp(DockerSwift dockerSwift) {
         this.dockerSwift = dockerSwift;
-        bucketName = BucketName.of(UUID.randomUUID().toString());
+        bucketName = BucketName.DEFAULT;
```

Comment:
I believe this was ensuring test isolation. Did you broke these tests?

As we can pass any bucket name we want to objectstorage implementation why can't we keep things as is - and just rename these fields as `defaultBucketName` ?